src/config.ts

```diff
@@ -147,4 +150,39 @@ export class ConfigManager
                 JSON.stringify(data, null, 4), this._defaultWriteOptions);
         }
     }
+    suggestedSettings(): any
+    {
+        return {
+            "C_Cpp.errorSquiggles": "Enabled",
```

Comment:
I don't believe the cosmetic changes have their place here. The whitespace stuff, intellisense and ruby lint, makes a lot of sense, but that and the debug config are really personal preference.

Reply:
OK.. Anything other than the error squiggles and debug.* ?

Reply:
I don't see anything else. The rest seem to be either functional settings, or cosmetic changes that are specific to Rock (the 4 space indent thing)

Reply:
Updated.